### PR TITLE
Rework deactivate account

### PR DESCRIPTION
**Pull request by YulichkaR**
### **Before** :
image

### **After** :
image

### CODE REVIEW

File: src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx [hunk 1]
```diff
@@ -1,227 +1,53 @@
...
-import useAxios from '~/hooks/use-axios'
-import { useAppDispatch, useAppSelector } from '~/hooks/use-redux'
-import useInputVisibility from '~/hooks/use-input-visibility'
-import useConfirm from '~/hooks/use-confirm'
-
-import { AuthService } from '~/services/auth-service'
-import { openAlert } from '~/redux/features/snackbarSlice'
-
-import { styles } from '~/containers/edit-profile/password-security-tab/PasswordSecurityTab.styles'
+import PasswordSecurityItem from './password-security-item/PasswordSecurityItem'
```
Comment (Olenka-Hryk) [inline src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx:7]: Can you organize the imports and use an alias `~`?

File: src/containers/edit-profile/password-security-tab/change-password-modal/ChangePasswordModal.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+import useInputVisibility from '~/hooks/use-input-visibility'
+import useConfirm from '~/hooks/use-confirm'
+
+import Loader from '~/components/loader/Loader'
+import AppButton from '~/components/app-button/AppButton'
+import AppTextField from '~/components/app-text-field/AppTextField'
+
+import { snackbarVariants } from '~/constants'
+
+import { useAppDispatch, useAppSelector } from '~/hooks/use-redux'
```
Comment (Olenka-Hryk) [inline src/containers/edit-profile/password-security-tab/change-password-modal/ChangePasswordModal.tsx:19]: Can you group imports by type?

File: src/containers/edit-profile/password-security-tab/password-security-item/PasswordSecurityItem.tsx
Comment (Olenka-Hryk) [inline src/containers/edit-profile/password-security-tab/password-security-item/PasswordSecurityItem.tsx:1]: Could you please add tests for `PasswordSecurityItem` container?

**Review by Olenka-Hryk — CHANGES_REQUESTED** [review]

File: src/containers/edit-profile/password-security-tab/password-security-item/PasswordSecurityItem.tsx
Comment (YulichkaR) [reply src/containers/edit-profile/password-security-tab/password-security-item/PasswordSecurityItem.tsx:1]: added

File: src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx (hunk 1 above)
Comment (YulichkaR) [reply src/containers/edit-profile/password-security-tab/PasswordSecurityTab.tsx]: organized

**Review by abalanovsky — APPROVED** [review]

**Review by docia — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

**Review by Olenka-Hryk — APPROVED** [review]

**Pull request merged**